spec/features/state_file/complete_intake_spec.rb

```diff
@@ -47,15 +49,15 @@
       end
       click_on I18n.t("general.continue")
 
-      expect(page).to have_text I18n.t("state_file.questions.nyc_residency.edit.title", year: 2023)
-      choose "I did not live in New York City at all in 2023"
+      expect(page).to have_text I18n.t("state_file.questions.nyc_residency.edit.title", year: 2024)
+      choose "I did not live in New York City at all in 2024"
```

Comment:
should this be `year: filing_year` etc?

Reply:
oh i guess on closer inspection it's just NY so i guess i don't care but it does feel weird that we're making tons of updates to NY stuff when we should in theory be deleting it * sigh *

Reply:
i just updated them to be consistent 🤷